### ISSUE TITLE
Browser crashing on initialization for element; maybe SaxParser related.

### ISSUE THREAD

**Issue by justmatt** [tinymce/tinymce#3534]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Crashing browser
Maybe related to SaxParser ![Screenshot](https://cloud.githubusercontent.com/assets/426173/23802511/84a8f318-05ab-11e7-97db-eb3400c00d7d.png)

_Steps to recreate:_
- Open [fiddle.tinymce.com](http://fiddle.tinymce.com/)
- Replace `<textarea name="content"></textarea>` with
```
<textarea name="content">
A monograph stated protocols for infusion of oxytocin vary by institution, but should include either administration of small, slow bolus doses (ie, <3 units IV), or controlled infusion. At one author’s institution, oxytocin 18 milliunits/minute IV was administered by controlled infusion, started after cord clamping, and titrated as needed for bleeding up to 36 milliunits/minute. The authors also highlighted the fact that the optimal dose and route of administration (ie, bolus dose versus infusion) of oxytocin at cesarean delivery were unclear. [5]

</textarea>
```

**What is the expected behavior?**
Should not crash

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
TinyMCE v4.0 & v4.5
Chrome v56.0.2924.87 (64-bit)
Firefox v34.0.5
Mac OSX 10.12.3 (Sierra)

**Comment by fyrkant** [comment]
It's because of the `<3` being interpreted as a html tag. We should try ti fix this soon, but for the moment: if you add a space between the two characters it will work `< 3`. 

related to #3047 #3300 #3436 #3463

**Comment by lnewson** [comment]
Hi,

I was able to reproduce this in TinyMCE 4.5.11, however I'm unable to reproduce it in the latest version of TinyMCE. Here's a fiddle to show it working: http://fiddle.tinymce.com/Qghaab

It's likely this has been fixed somewhere between 4.5.11 and 4.9.10, so I'm going to close this issue. If you're still able to reproduce this in the latest version however, then let us know and we'll reopen this issue.

Thanks!
